### PR TITLE
[zh-cn]: update the translation of ArrayBuffer.slice()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#16958

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md [hunk 1]
```diff
@@ -5,38 +5,43 @@ slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
...
 ```
 
 ### 参数
 
-- `begin`
-  - : 从零开始的字节索引，切片从这开始。
-- `end`
-  - : 结束切片的字节索引。如果没指定 end，新的 `ArrayBuffer` 将包含这个 `ArrayBuffer` 从头到尾的所有字节。由 begin 和 end 指定的这个范围夹在当前数组的有效索引范围内。如果新`ArrayBuffer`的长度在计算后为负，它将强制为 0。
+- `start` {{optional_inline}}
+  - : 要开始提取的位置下标，以 0 为起始，将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md:23]:
```suggestion
  - : 要开始提取的位置索引（从 0 开始），将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。
```

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md [hunk 2]
```diff
@@ -5,38 +5,43 @@ slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
...
+    - 负数下标将会从缓冲区末尾开始计算——如果 `start < 0`，那么将会使用 `start + buffer.length`。
+    - 如果 `start < -buffer.length` 或省略了 `start`，则会使用 `0`。
+    - 如果 `start >= buffer.length`，则不会提取任何内容。
+- `end` {{optional_inline}}
+  - : 要结束提取的位置下标，以 0 为起始，将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。`slice()` 提取到但不包括 `end`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md:28]:
```suggestion
  - : 要结束提取的位置索引（从 0 开始），将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。`slice()` 提取到但不包括 `end`。
```

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md [hunk 3]
```diff
@@ -5,38 +5,43 @@ slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
...
 
 ### 参数
 
-- `begin`
-  - : 从零开始的字节索引，切片从这开始。
-- `end`
-  - : 结束切片的字节索引。如果没指定 end，新的 `ArrayBuffer` 将包含这个 `ArrayBuffer` 从头到尾的所有字节。由 begin 和 end 指定的这个范围夹在当前数组的有效索引范围内。如果新`ArrayBuffer`的长度在计算后为负，它将强制为 0。
+- `start` {{optional_inline}}
+  - : 要开始提取的位置下标，以 0 为起始，将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。
+    - 负数下标将会从缓冲区末尾开始计算——如果 `start < 0`，那么将会使用 `start + buffer.length`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md:24]:
```suggestion
    - 负数索引将会从缓冲区末尾开始计算——如果 `start < 0`，那么将会使用 `start + buffer.length`。
```

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md [hunk 4]
```diff
@@ -5,38 +5,43 @@ slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
...
+    - 如果 `start < -buffer.length` 或省略了 `start`，则会使用 `0`。
+    - 如果 `start >= buffer.length`，则不会提取任何内容。
+- `end` {{optional_inline}}
+  - : 要结束提取的位置下标，以 0 为起始，将被[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。`slice()` 提取到但不包括 `end`。
+    - 负数下标将会从缓冲区末尾开始计算——如果 `end < 0`，那么将会使用 `end + buffer.length`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md:29]:
```suggestion
    - 负数索引将会从缓冲区末尾开始计算——如果 `end < 0`，那么将会使用 `end + buffer.length`。
```

**Review by yin1999 — COMMENTED** [review]
Almost looks good. But could we use "索引" which should be formaler.

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t. One more thing, could you also add the [`l10n.sourceCommit`](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE) metadata? We may use it in the feature.

**Comment by JasonLamv-t** [comment]
> Thank you @JasonLamv-t. One more thing, could you also add the [`l10n.sourceCommit`](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE) metadata? We may use it in the feature.

Sure. I will update this PR later.

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md [hunk 5]
```diff
@@ -1,42 +1,49 @@
 ---
 title: ArrayBuffer.prototype.slice()
 slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
+l10n:
+  sourceCommit: d254545601e07aa48a6bb9d5bfc1a75f6561de4d
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md:5]: Please use the commit sha from [the corresponding file](https://github.com/mdn/content/commits/main/files/en-us/web/javascript/reference/global_objects/arraybuffer/slice/index.md)

```suggestion
  sourceCommit: 27180875516cc311342e74b596bfb589b7211e0c
```

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md]: Got it.

**Pull request merged**